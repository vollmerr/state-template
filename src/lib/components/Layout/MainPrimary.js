import React from 'react';

import LayoutBase from './LayoutBase';

const MainPrimary = (props) => (
  <LayoutBase tag={'main'} baseClass={'main-primary'} {...props} />
);

export default MainPrimary;
