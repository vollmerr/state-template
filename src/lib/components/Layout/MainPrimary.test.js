import React from 'react';
import { shallow } from 'enzyme';

import { MainPrimary as fromLib } from '../..';
import MainPrimary from './MainPrimary';

const props = {
  children: 'test-children',
  className: 'test-class',
};

let wrapper;
describe('MainPrimary', () => {
  beforeEach(() => {
    wrapper = shallow(<MainPrimary {...props} />);
  });

  it('should be exported as part of the library', () => {
    expect(fromLib).toEqual(MainPrimary);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
