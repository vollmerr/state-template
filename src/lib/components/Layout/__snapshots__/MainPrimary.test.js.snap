// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPrimary should render correctly 1`] = `
<LayoutBase
  baseClass="main-primary"
  className="test-class"
  tag="main"
>
  test-children
</LayoutBase>
`;
